src/BornOnThisDayApi.java

```diff
@@ -15,30 +15,35 @@
 
     /**
      * BornOnThisDay.getResponse()
-     * Sends HTTP request for 'births' to Wikimedia's 'On this day' api. The response is a JSON object,
+     * Sends HTTP request for 'births' to Wikimedia's 'On this day' api. The response is a JSON object
      * with a value of a JSON Array representing people born on the given day.
      * The response is read in and converted to a String
      *
-     * @param mm a two-character String representing the two digit month given by the user to be appended
+     * @param mm a two-digit integer representing the two-digit month given by the user to be parsed as a String and appended
      *           to the url path.
-     * @param dd a two character String representing the two digit day given by the user to be appended
-     *           to the url path
+     *
+     * @param dd a two-digit integer representing the two-digit day given by the user to be parsed as a String and appended
+     *           to the url path.
+     *
      * @return response.toString()
+     *
      * @author Bradley Boggs, bradleywboggs@gmail.com
      */
 
-    public static String getResponse(String mm, String dd) {
+    public static String getResponse(int mm, int dd) {
+
         try{
+
             String url = String.format("https://en.wikipedia.org/api/rest_v1/feed/onthisday/births/%s/%s", mm, dd);
-            //Instantiated URL object
+            // Instantiate URL object
```

Comment:
Consider not having comments where the code is already obvious about what is happening.